src/Kiota.Builder/Plugins/PluginsGenerationService.cs

```diff
@@ -88,6 +88,14 @@ public async Task GenerateManifestAsync(CancellationToken cancellationToken = de
                 case PluginType.APIPlugin:
                     var pluginDocument = GetManifestDocument(descriptionRelativePath);
                     pluginDocument.Write(writer);
+
+                    var shouldGenerateTemplate = pluginDocument.Functions.Any(static x => x.Capabilities?.ResponseSemantics?.StaticTemplate is JsonElement template && template.TryGetProperty("file", out _));
+                    if (shouldGenerateTemplate)
+                    {
+                        var adaptiveCardOutputPath = Path.Combine(Configuration.OutputPath, "adaptive-card.json");
```

Comment:
please, change the name of the generated adaptive card to be the same as the function name per Seb requests in the issue :)